4ds.bt

```diff
@@ -360,36 +368,52 @@ struct OCCLUDER
     FACE faces[numFaces];
 };
 
+struct PROJECTOR
```

Comment:
Out of curiosity, which models contain these frame types?

Reply:
They are featured in Wings of War and Circus Empire, there are quite a lot of them
FRAME_PROJECTOR
\Circus Empire\Models\Interiors_efects\eff_model_svetla-vzor1.4ds

FRAME_EMITOR
\Wings Of War\models\Arras_prepady.4ds

FRAME_LIGHT and VISUAL_LITOBJECT
\Circus Empire\Models\Buildings\Exteriors\j_bigboard.4ds

These are just a few, I can compile a full list.

Reply:
Ah. I see! I thought there were some of them in Mafia :)